prime-router/settings/prod/0009-nj-doh-elr.yml

```diff
@@ -0,0 +1,16 @@
+# NJ DOH ELR connection
+name: elr
+organizationName: nj-doh
+topic: covid-19
+jurisdictionalFilter:
+  - matches(ordering_facility_state, NJ)
+translation:
+  type: HL7
+  useBatchHeaders: true
```

Comment:
Good.

Just an FYI while I'm thinking of it, my load testing has shown that the singleton HL7 (without batching) is really bad for performance.   Not just the many sftps but the many database transactions - its just really slow.    So try to avoid it whenever possible.